indra/newview/llinventorybridge.cpp

```diff
@@ -5368,32 +5373,65 @@ void LLFolderBridge::createWearable(LLFolderBridge* bridge, LLWearableType::ETyp
     LLAgentWearables::createWearable(type, false, parent_id);
 }
 
-void LLFolderBridge::modifyOutfit(BOOL append)
+//<FS:AR> FIRE-31508: refactored from void LLFolderBridge::modifyOutfit(BOOL append)
+bool LLFolderBridge::modifyOutfitExceedsWearFolderLimit()
 {
-    LLInventoryModel* model = getInventoryModel();
-    if(!model) return;
-    LLViewerInventoryCategory* cat = getCategory();
-    if(!cat) return;
+    LLInventoryModel *model = getInventoryModel();
```

Comment:
I don't know what this is, doesn't appear used... 
I have no idea what state it creates (if any). It's in the old code, so I carried it around.

Reply:
This is either obsolete or `model` should be used instead of `gInventoryModel` to collect the category contents a few line further down.

Reply:
`model` appears vestigial...
`getCatagory()` itself performs a `getInventoryModel()` for its interrogation... it might be a hangover?

`getCategory()` will yield null if `getInventoryModel()` yields null, so appears safe enough to comment out.